src/core/server/legacy/legacy_service.ts

```diff
@@ -267,6 +267,7 @@ export class LegacyService implements CoreService {
         setClientFactoryProvider: setupDeps.core.savedObjects.setClientFactoryProvider,
         addClientWrapper: setupDeps.core.savedObjects.addClientWrapper,
         registerType: setupDeps.core.savedObjects.registerType,
+        registerExportHook: setupDeps.core.savedObjects.registerExportHook,
```

Comment:
nit: instead of having a dedicated `registerExportHook`, should we make it part of the existing `SavedObjectsTypeManagementDefinition` interface since this already has `importableAndExportable`?

Reply:
Yea, we came with the same conclusion with @rudolf in https://github.com/elastic/kibana/issues/84980#issuecomment-759473258. I will adapt that, as I did in https://github.com/elastic/kibana/pull/87996